cmd/txtar-addmod/addmod.go

```diff
@@ -44,9 +44,13 @@ func fatalf(format string, args ...interface{}) {
 	log.Fatalf(format, args...)
 }
 
-const goCmd = "vgo"
+const goCmd = "go"
 
 func main() {
+	os.Exit(mainint())
+}
+
+func mainint() int {
```

Comment:
Not so keen on "mainint" as a name. I think I'd prefer main1 or Main if that's OK.

Also, maybe add a TODO to avoid calling log.Fatalf (that'll mean we can't get test coverage in those circumstances)

Reply:
Went with `main1` per your suggestion. I think that's a good convention.